css-animations-1/Overview.bs

```diff
@@ -877,9 +877,9 @@ The <code>AnimationEvent</code> Interface</h3>
 IDL Definition</h4>
 
 	<pre class="idl">
-		[Exposed=Window,
-		 Constructor(CSSOMString type, optional AnimationEventInit animationEventInitDict = {})]
+		[Exposed=Window]
 		interface AnimationEvent : Event {
+		  constructor(CSSOMString type, optional AnimationEventInit animationEventInitDict = {});
```

Comment:
Can we fix the link later in this section to be:

```
<dfn constructor for=AnimationEvent>AnimationEvent(type, animationEventInitDict)</dfn>
```

as well so it links correctly?